.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2
 aliases:
   - &defaults
     docker:
-      - image: circleci/ruby:2.6-stretch-node
+      - image: circleci/ruby:2.6-buster-node
```

Comment:
what's the difference between stretch and buster?

Reply:
https://www.debian.org/releases/stretch/

> Debian 9 has been superseded by Debian 10 ("buster"). Security updates have been discontinued as of July 6th, 2020.

Reply:
Gargron, thanks for your explanation!